src/ngSanitize/sanitize.js

```diff
@@ -200,6 +200,44 @@ function $SanitizeProvider() {
     }
   };
 
+
+  /**
+   * @ngdoc method
+   * @name $sanitizeProvider#addValidElements
+   * @kind function
+   *
+   * @param {Array|Object=} elements List of valid elements.
+   */
+  this.addValidElements = function(elements) {
+    var _validElements = elements;
+
+    if (angular.isObject(elements)) {
+      _validElements = [];
+      angular.forEach(['voidElements', 'inlineElements', 'blockElements', 'svgElements'], function(elementsType) {
+        if (elements[elementsType]) {
+          if (elementsType === 'voidElements') {
+            angular.extend(voidElements, toMap(elements.voidElements.join(',')));
+          }
+          _validElements = _validElements.concat(elements[elementsType]);
+        }
+      });
+    }
+
+    angular.extend(validElements, toMap(_validElements.join(',')));
+  };
+
+
+  /**
+   * @ngdoc method
+   * @name $sanitizeProvider#addValidAttrs
+   * @kind function
+   *
+   * @param {Array} attrs List of valid attributes.
+   */
```

Comment:
It might be worth documenting that the added attributes will not be treated as URI attributes, i.e. their values will not be sanitized as URIs.